.github/workflows/ci.yml

```diff
@@ -29,11 +29,11 @@ jobs:
       - name: build odin
         run: make release
       - name: Odin run
-        run: |
-          ./odin run examples/demo/demo.odin
-          ./odin run examples/demo/demo.odin -llvm-api
+        run: ./odin run examples/demo/demo.odin
       - name: Odin check
         run: ./odin check examples/demo/demo.odin -vet
+      - name: Odin version
```

Comment:
Why do you want this extra step to run? It's just extra time spend in CI unless you plan on going and checking it?